themes/ddbasic/sass/components/header.scss

```diff
@@ -3,702 +3,451 @@
-      }
-      // Search Form Extended with admin menu shortcuts or toolbar
-      .search-form-extended.admin-menu-with-shortcuts &,
-      .search-form-extended.toolbar & {
-        top: $toolbar-height - 20px;
-      }
-      // Search Form Extended on frontpage
-      .search-form-extended.page-search &,
-      .search-form-extended.front & {
-        top: $topbar-height;
+  // Search form extended with profiles
+  .search-form-extended-with-profiles & {
+    .pane-search-form {
+      .form-item-search-block-form {
+        width: getColumn(8) + getGutter();
 
         // Tablet
         @include media($tablet) {
-          top: $header-height;
+          width: 100%;
+          padding-right: 270px;
         }
-      }
-      // Search Form Extended on frontpage with admin-menu
-      .search-form-extended.page-search.admin-menu &,
-      .search-form-extended.front.admin-menu & {
-        top: $topbar-height + $admin-menu-height;
-      }
-      // Search Form Extended on frontpage with admin menu shortcuts or toolbar
-      .search-form-extended.page-search.admin-menu-with-shortcuts &,
-      .search-form-extended.front.admin-menu-with-shortcuts &,
-      .search-form-extended.page-search.toolbar &,
-      .search-form-extended.front.toolbar &  {
-        top: $topbar-height + $toolbar-height;
-      }
-      // Extended search open
-      .extended-search-is-open & {
-        top: $topbar-height;
-        box-shadow: $box-shadow;
-      }
-      // Extended search open with admin menu
-      .extended-search-is-open.admin-menu & {
-        top: $topbar-height + $admin-menu-height;
-      }
-      // Extended search open with admin menu shortcuts or toolbar.
-      .extended-search-is-open.admin-menu-with-shortcuts &,
-      .extended-search-is-open.toolbar & {
-        top: $topbar-height + $toolbar-height;
-      }
-      // Search form extended front
-      .search-form-extended.page-search &,
-      .search-form-extended.front & {
-        box-shadow: $box-shadow;
-        .search-extended-button {
-          display: none;
+
+        // Mobile
+        @include media($mobile) {
+          padding-right: 70px;
         }
+      }
+      .form-item-ting-field-search {
+        width: getColumn(3) + getGutter();
+        float: left;
+        margin-bottom: 0;
 
         // Tablet
         @include media($tablet) {
-          box-shadow: none;
+          position: absolute;
+          top: 0;
+          right: 70px;
+          width: 200px;
+        }
+
+        // Mobile
+        @include media($mobile) {
+          position: static;
+          width: 100%;
+          margin-top: 10px;
         }
       }
-      // Search form extended front when mobile search is open
-      .search-form-extended.page-search.mobile-search-is-open &,
-      .search-form-extended.front.mobile-search-is-open & {
-        // Tablet
-        @include media($tablet) {
-          box-shadow: $box-shadow;
+      .select-wrapper {
```

Comment:
This looks like a custom class to me. The name is very generic. Is the a reason why we cannot use something like `.search-form-extended-with-profiles-select-wrapper`?

Reply:
All select lists are wrapped in a div with the class .select-wrapper. This is done for general styling. Here we need to overwrite a few of these general stylings.